roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -17,6 +17,27 @@
 - name: Get ceph_cli
   ansible.builtin.include_tasks: ceph_cli.yml
 
+- name: Check if swift endpoint is already created
+  shell: "oc rsh openstackclient openstack endpoint list | grep 'swift.*object-store' | wc -l"
+  register: swift_endpoint
+  delegate_to: localhost
+
+- name: Configure ceph object store to use external ceph object gateway
+  cifmw.general.ci_script:
+    output_dir: "/home/zuul/ci-framework-data/artifacts"
+    script: |-
+      oc rsh  openstackclient openstack service create --name swift --description 'OpenStack Object Storage' object-store
```

Comment:
The last failure is weird:

```
2024-02-12 07:02:49.908197 | controller | TASK [cifmw_cephadm : Configure ceph object store to use external ceph object gateway output_dir=/home/zuul/ci-framework-data/artifacts, script=oc rsh  openstackclient openstack service create --name swift --description 'OpenStack Object Storage' object-store
2024-02-12 07:02:49.908218 | controller | oc rsh  openstackclient openstack user create --project service --password {{ cifmw_ceph_rgw_keystone_psw }}  swift
2024-02-12 07:02:49.908223 | controller | oc rsh  openstackclient openstack role create swiftoperator
2024-02-12 07:02:49.908238 | controller | oc rsh  openstackclient openstack role create ResellerAdmin
2024-02-12 07:02:49.908242 | controller | oc rsh  openstackclient openstack role add --user swift --project service member
2024-02-12 07:02:49.908247 | controller | oc rsh  openstackclient openstack role add --user swift --project service admin
2024-02-12 07:02:49.908252 | controller | oc rsh  openstackclient openstack endpoint create --region regionOne object-store public http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
2024-02-12 07:02:49.908262 | controller | oc rsh  openstackclient openstack endpoint create --region regionOne object-store internal http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
2024-02-12 07:02:49.908267 | controller | oc rsh  openstackclient openstack role add --project admin --user admin swiftoperator] ***
2024-02-12 07:02:49.908271 | controller | Monday 12 February 2024  07:02:49 -0500 (0:00:00.331)       0:33:53.693 *******
2024-02-12 07:02:50.123385 | controller | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: NoneType: None
2024-02-12 07:02:50.123426 | controller | fatal: [compute-0 -> localhost]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 1, "stderr": "", "stderr_lines": [], "stdout": "error: Missing or incomplete configuration info.  Please point to an existing, complete config file:\n\n\n  1. Via the command-line flag --kubeconfig\n  2. Via the KUBECONFIG environment variable\n  3. In your home directory as ~/.kube/config\n\nTo view or setup config directly use the 'config' command.\n", "stdout_lines": ["error: Missing or incomplete configuration info.  Please point to an existing, complete config file:", "", "", "  1. Via the command-line flag --kubeconfig", "  2. Via the KUBECONFIG environment variable", "  3. In your home directory as ~/.kube/config", "", "To view or setup config directly use the 'config' command."]}
```

so it's like it delegates the tasks to a node where KUBECONFIG is not set.